addons/hivedr/README

```diff
@@ -69,12 +68,9 @@ a. Perform initial bootstrap of Table and Database from one Hadoop cluster to an
 b. Setup cluster definition
    $FALCON_HOME/bin/falcon entity -submit -type cluster -file /cluster/definition.xml
 
-c. Submit Hive DR recipe
-   $FALCON_HOME/bin/falcon recipe -name hive-disaster-recovery -operation HIVE_DISASTER_RECOVERY
+c. Submit Hive mirroring extension
+   $FALCON_HOME/bin/falcon extension -submitAndSchedule -extensionName hive-mirroring -file /process/definition.xml
```

Comment:
@bvellanki : Detailed documentation will be added by Ying as part of FALCON-1904. I don't think the information should be repeated here as I am already providing the reference to CLI or REST API doc which will be updated as part of FALCON-1904. Same is done for cluster submit if you look above.